spartan/devnet-smoke-tests/create_new_accounts.sh

```diff
@@ -0,0 +1,34 @@
+# Create new accounts
+new_accounts_per_run=2
+new_accounts="[]"
+max_accounts=2
+
+total_account_count=$(jq '.accounts | length' state.json)
+if [ "$total_account_count" -lt "$max_accounts" ]; then
+
+  # New accounts should be the min(num_new_accounts, max_accounts - total_account_count)
+  num_new_accounts=$(( new_accounts_per_run < (max_accounts - total_account_count) ? new_accounts_per_run : (max_accounts - total_account_count) ))
+  for i in $(seq 1 $num_new_accounts); do
+    new_account=$(aztec-wallet create-account -a main --register-only --node-url $NODE_URL --json | grep -Pzo '(?s)\{.*\}')
+    new_account=$(echo "$new_account" | jq '. + {"inited": false}')
+
+    echo "$new_account" | jq '.'
+
+    aztec-wallet bridge-fee-juice 99900000000000000000 main --mint --node-url $NODE_URL --l1-rpc-urls http://35.233.242.32:8545 --l1-chain-id 1337
+
+    # waits two blocks for l2 message to become available
+    sleep 1.5m
```

Comment:
Why is 1.5m two blocks? Also you should at least output a message if you're going to shutdown for almost two minutes.